docs/get-started.md

```diff
@@ -1,4 +1,3 @@
-## Get Started
```

Comment:
I find the other `.md` file's name is upper case . Please rename `get-started.md` to `Get-started.md` thanks

Reply:
If the filename will be showed in the address bar of the browser, lower case might be more suitable, except some special ones, like `nGQL`.

Reply:
fixed